STYLE.md

```diff
@@ -18,6 +18,7 @@ sentence or heading.
 - taproot
 - merkle (tree|root|branch|node)
 - segwit
+- bech32
```

Comment:
Should these be sorted as with "Forbidden abbreviations" and "Compound words" below?

Reply:
I didn't comment on this because the list wasn't sorted before this PR. @harding , was there a reason for that?

Reply:
No reason.  I must've forgotten to do it (I typed things as they occurred to me and then sorted by piping ranges from my editor to `sort`).